src/components/Header/Header.module.sass

```diff
@@ -0,0 +1,37 @@
+.header
+    display: flex
+    flex-direction: row
+    padding: 18px
+
+.headerLeftSide
+    display: flex
+    align-items: center
+
+.headerRightSide
+    display: flex
+    flex-direction: row
+    align-items: center
+    justify-content: space-between
+    width: 100%
+    padding-left: 10px
+    padding-right: 3px
+
+.userAndAvatar
+    display: flex
+    align-items: center
+    gap: 13px
+
+.companyLogo
```

Comment:
we can refactor the styling so that it won't be redundant

```
.companyLogo,
.menuButton,
.cartButton
value here
```

will this work?

Reply:
That is a better way to do it. Refactored and it works.